src/derivatives/atlas.md

```diff
@@ -97,8 +95,8 @@ A guide for using macros can be found at
    "colin27-pipeline": {
       "sub-01": {
          "anat": {
-            "sub-01_label-brain_mask.nii.gz": "",
-            "sub-01_label-head_mask.nii.gz": "",
+            "sub-01_seg-brain_mask.nii.gz": "",
+            "sub-01_seg_head_mask.nii.gz": '',
```

Comment:
This is plain BIDS derivatives, doesn't pertain to this BEP. Currently, masks have `label`, unless I am missing something or something changed while I was not looking.

```suggestion
            "sub-01_label-brain_mask.nii.gz": "",
            "sub-01_label-head_mask.nii.gz": "",
```